File: 01_Practice_Lec_Codes/DecimalToBinary.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main (){
    int n = 4;  
    cout << "Enter n";
   

    int i = 0;
    int ans = 0;
    while (n!=0)
    {
        int bit = n&1;
        ans = (bit*pow(10,i)) + ans;
        n>>1;
        i++;
    }
    cout << "The answer in binary is "<< ans << endl;
    
}

File: 01_Practice_Lec_Codes/DecimalToBinary.java

import java.util.Scanner;

public class DecimalToBinary {

    public static void main(String[] args) {
     Scanner sc = new Scanner(System.in);
     System.out.println("Enter the number");
     int n = sc.nextInt();
     sc.close();
       int ans = 0;
     int i =0;
     int num = 10;
     while (n!=0) {
        int Bit = n & 1;
         ans = Bit*(int) ((Math.pow(num, i))) + ans;
        n = n>>1;
        i++;
     }
    
     System.out.println(ans);


    }
    }

    

